azureDatabaseUtils/gradle/dependencies.gradle

```diff
@@ -3,6 +3,8 @@ dependencies {
   // Azure dependencies
   implementation 'com.azure:azure-identity:1.9.1'
   implementation 'com.azure:azure-identity-extensions:1.1.4'
+  implementation 'org.postgresql:postgresql'
```

Comment:
this is already on line 9 below

Reply:
thanks - I didn't recognize it as being equivalent